### DISCUSSION TITLE
Specify --iglob-file  or  --glob-file in configuration

### DISCUSSION THREAD

**Discussion by HaleTom** [rustic-rs/rustic#1071] [Q&A] [answered]
Is there a way to specify `--iglob-file`  or  `--glob-file` inside config to reduce command line length?

**Comment by aawsome** [comment, marked as answer]
see https://github.com/rustic-rs/rustic/blob/main/config/full.toml#L89

**Comment by HaleTom** [comment]
Looking at the indicated line, this seems to be an array of exclude entries.  I'm looking for a place to specify the filename to avoid specfying it at the command line.

Or is it an array of --iglob-file style fileNAMES?

**Reply by aawsome** [reply]
Yes, array of filenames:
```
iglob-file = ["/path/to/my-exclude-file1.txt", "/path/to/my-exclude-file2.txt"]
```
Can be of course an array with only 1 entry.

(we should add the possibility to also use `iglob-file = "/path/to/my-exclude-file1.txt"` but this needs to be implemented)
